lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
+#: ultralcd.cpp:1985
+msgid "Date:"
+msgstr "Datum:"
+
+# MSG_COMMUNITY_MADE c=18
+#: messages.c:25
+msgid "Community made"
+msgstr "Allm\xe1nhetsgjord"
+
+# MSG_DISABLE_STEPPERS c=18
+#: ultralcd.cpp:5678
+msgid "Disable steppers"
+msgstr "Inaktivera stepper"
+
+# MSG_BABYSTEP_Z_NOT_SET c=20 r=12
+#: messages.c:13
+msgid "Distance between tip of the nozzle and the bed surface has not been set yet. Please follow the manual, chapter First steps, section First layer calibration."
+msgstr "Avstandet mellan munstyckets spets och b\xe1ddytan har inte st\xe1llts in \xe1nnu. V\xe1nligen f\xeflj manualen F\xefrsta lagrets kalibrering."
+
+# MSG_FS_CONTINUE c=5
+#: messages.c:152
+msgid "Cont."
+msgstr "Frts."
+
+# MSG_WIZARD_REPEAT_V2_CAL c=20 r=7
+#: ultralcd.cpp:5021
+msgid "Do you want to repeat last step to readjust distance between nozzle and heatbed?"
+msgstr "Vill du upprepa det sista steget f\xefr att justera avstandet mellan munstycket och v\xe1rmeb\xe1edden?"
+
+# MSG_EXTRUDER_CORRECTION c=13
+#: ultralcd.cpp:5090
+msgid "E-correct:"
+msgstr "E-korrektion:"
+
+# MSG_EJECT_FILAMENT c=16
+#: messages.c:60
+msgid "Eject filament"
+msgstr "Mata ut filament"
+
+# MSG_EJECTING_FILAMENT c=20
+#: mmu.cpp:1415
+msgid "Ejecting filament"
+msgstr "Matar ut filament"
+
+# MSG_SELFTEST_ENDSTOP_NOTHIT c=20
+#: ultralcd.cpp:8149
+msgid "Endstop not hit"
+msgstr "\xe1ndlage inte natt"
+
+# MSG_SELFTEST_ENDSTOP c=16
+#: ultralcd.cpp:8144
+msgid "Endstop"
+msgstr "\xe1ndl\xe1ge"
+
+# MSG_SELFTEST_ENDSTOPS c=20
+#: ultralcd.cpp:8135
+msgid "Endstops"
+msgstr "\xe1ndl\xe1gen"
+
+# MSG_STACK_ERROR c=20 r=4
+msgid "Error - static memory has been overwritten"
+msgstr "Fel - statiskt minne har skrivits \xefver"
+
+# MSG_CUT_FILAMENT c=16
+#: messages.c:61
+msgid "Cut filament"
+msgstr "Sk\xe1r filament"
+
+# MSG_CUTTER c=9
+#: messages.c:128
+msgid "Cutter"
+msgstr "Sk\xe1rare"
+
+# MSG_MMU_CUTTING_FIL c=18
+#: mmu.cpp:1387
+msgid "Cutting filament"
+msgstr "Sk\xe1r filament"
+
+# MSG_FSENS_NOT_RESPONDING c=20 r=4
+#: ultralcd.cpp:4330
+msgid "ERROR: Filament sensor is not responding, please check connection."
+msgstr "FEL: Filamentsensorn svarar inte, kontrollera anslutningen."
+
+# MSG_DIM c=6
+#: messages.c:159
+msgid "Dim"
+msgstr "Dim"
+
+# MSG_ERROR c=10
+#: messages.c:31
+msgid "ERROR:"
+msgstr "FEL:"
+
+# MSG_SELFTEST_EXTRUDER_FAN_SPEED c=18
+#: ultralcd.cpp:8498
+msgid "Extruder fan:"
+msgstr "Extruderfl\xe1kt:"
+
+# MSG_INFO_EXTRUDER c=18
+#: ultralcd.cpp:2040
+msgid "Extruder info"
+msgstr "Extruder info"
+
+# MSG_EXTRUDER c=17
+#: messages.c:32
+msgid "Extruder"
+msgstr "Extruder"
+
+# MSG_MMU_FAIL_STATS c=18
+#: ultralcd.cpp:6754
+msgid "Fail stats MMU"
+msgstr "Felstatistik MMU"
+
+# MSG_FSENSOR_AUTOLOAD c=13
+#: messages.c:48
+msgid "F. autoload"
+msgstr "F. autoladdn"
+
+# MSG_FAIL_STATS c=18
+#: ultralcd.cpp:6751
+msgid "Fail stats"
+msgstr "Felstatistik"
+
+# MSG_FAN_SPEED c=14
+#: messages.c:36
+msgid "Fan speed"
+msgstr "Fl\xe1ktfart"
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.